spec/system/distribution_system_spec.rb

```diff
@@ -555,10 +555,12 @@
       expect(page).to have_css("table tbody tr td", text: stored_item1_total)
     end
 
+    # TODO: figure this out
+    # rspec ./spec/system/distribution_system_spec.rb[1:11:2,1:11:4] --seed 58495
     it "filters by item category id" do
       @organization.item_categories << item_category
-      create(:distribution, :with_items, item: item1)
-      create(:distribution, :with_items, item: item2)
+      create(:distribution, :with_items, item: item1, organization: @organization)
+      create(:distribution, :with_items, item: item2, organization: @organization)
```

Comment:
No clue why this one is failing either. `@organization` has the correct categories but when it goes into the controller is loses them, not clue why and it is order dependent.